torch/utils/_pytree.py

```diff
@@ -170,16 +172,16 @@ class _SerializeNodeDef(NamedTuple):
 # NB: we try really hard to not import _cxx_pytree (which depends on optree)
 # as much as possible. This is for isolation: a user who is not using C++ pytree
 # shouldn't pay for it, and it helps makes things like cpython upgrades easier.
+_optree_minimum_version = _TorchVersion("0.13.0")
```

Comment:
NB: `TorchVersion` is a subclass of `str` while the rich comparison is semantic version comparison rather than alphabetic dictionary order.